csharp_dotnetcore/13.Basic-Bot-Template/Dialogs/Greeting/GreetingDialog.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.BotBuilderSamples
     /// <summary>
     /// Demonstrates the following concepts:
     /// - Use a subclass of ComponentDialog to implement a mult-turn conversation
-    /// - Use a Waterflow dialog to model multi-turn conversation flow
+    /// - Use a Waterfall dialog to model multi-turn conversation flow
```

Comment:
For `"Waterfall dialog"` it should be `"WaterfallDialog"`, can you change this line to that and for all the other lines as well?